core/src/main/resources/swagger/rawls.yaml

```diff
@@ -3390,6 +3390,37 @@ paths:
               - email
               - profile
               - https://www.googleapis.com/auth/devstorage.full_control
+  '/api/workspaces/tags/autocomplete/{query}':
```

Comment:
In terms of making good REST APIs, I'd prefer that:
* the endpoint is a noun, not a verb: "tags" not "autocomplete"
* filters are query params

_(I like this page as a reference: http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api)_

So I suggest you move the endpoint to `/api/workspaces/tags?q={userinput}`

Thinking ahead to future stories, when we need an endpoint that returns only those tags **associated with workspaces to which the user has access**, this would be easily achievable through another query param, e.g. `all=(true|false)`. We could also, in the future, make the `q` param optional, as a means of returning all tags.